Userland/Libraries/LibWeb/CSS/CSSPropertyRule.idl

```diff
@@ -0,0 +1,11 @@
+#import <CSS/CSSRule.idl>
+
+// https://drafts.css-houdini.org/css-properties-values-api/#the-css-property-rule-interface
```

Comment:
This is the source references by MDN, but the host doesn't match some of the other links I've seen around

Reply:
`@property` is part of the Houdini project which is still considered experimental, so it's on its own domain. That's fine. :^)